lib/test/lib_test.rego

```diff
@@ -1,6 +1,3 @@
-# METADATA
-# custom:
-#   library: true
 package lib.test
```

Comment:
Would that get rid of the message that there are no selectors?

Reply:
Actually it might be just enough to rename the file as _test.rego based on my testing. I'll give it another try to confirm.

Reply:
It seems to be the case, I've reverted this back.